package-lock.json

```diff
@@ -0,0 +1,2947 @@
+{
```

Comment:
Please remove this file from the PR. It is unrelated to the addition of support for Solidity.